internal/p2p/peermanager.go

```diff
@@ -405,6 +405,13 @@ func (m *PeerManager) Add(address NodeAddress) (bool, error) {
 	if ok {
 		return false, nil
 	}
+	if peer.Inactive {
+		return false, nil
+	}
+
+	// set the peer's mutable score to something non-zero so that
+	// peer's we've never
+	peer.MutableScore = 256
```

Comment:
can we have this as a constant somewhere at the top

Reply:
I think this is probably meant to be `PeerScorePersistent`

Reply:
No, this is different - it's the default score we give to new peers (not persistent ones)